demos/image_classification_with_string_output/README.md

```diff
@@ -0,0 +1,74 @@
+# String output model demo {#ovms_string_output_model_demo}
+## Overview
+
+This demo demonstrates example deployment of a model with output precision `ov::element::string`. The output text is serialized into corresponding fields in gRPC proto/REST body. This allows the client to consume the text directly and avoid the process of label mapping or detokenization.
+
+### Download and prepare MobileNet model using from TensorFlow
+The script below is downloading a public MobileNet model trained on the ImageNet data. The original model accepts on input the image array in the range of 0-1 and returns probabilities for all the trained classes. We are adding to the model preprocessing function changing the input data range to 0-255 and also postprocessing function which is retrieving the most likely label name as a string. 
+This is a very handy functionality because if allows us to export the model with the included pre/post processing functions as the model layers. The client just receives the string data with the label name for the classified image.
```

Comment:
```suggestion
This is a very handy functionality because it allows us to export the model with the included pre/post processing functions as the model layers. The client just receives the string data with the label name for the classified image.
```